### DISCUSSION TITLE
Quick Reblog always "processing" but never reblogging

### DISCUSSION THREAD

**Discussion by rendavison** [AprilSylph/XKit-Rewritten#418] [Help Desk] [answered]
New bug that just cropped up today. When I reblog from the quick reblog window is says "processing" but never completes. If anyone could help me with troubleshooting I'd appreciate it.

**Comment by marcustyphoon** [comment, marked as answer]
Update to XKit Rewritten 0.14.2! Both Firefox and Chrome/Edge/Opera have buttons in their extension management interfaces to manually update, I believe; you may have to flip a "developer mode" switch or click a gear icon to find it.

**Reply by rendavison** [reply]
Thank you! That fixed it. For some reason Firefox didn't auto-update.

**Comment by arcanetrivia** [comment]
@rendavison I keep this setting turned off in Tools > Addons and Themes because I don't want auto updates on addons, but check it to see:

![updates](https://user-images.githubusercontent.com/68124115/151437564-2d7bde7a-1880-4aaa-9737-91b5114de48d.png)
